tests/run-custom-args/tasty-inspector/stdlibExperimentalDefinitions.scala

```diff
@@ -32,23 +32,6 @@ val experimentalDefinitionInLibrary = Set(
   "scala.annotation.MainAnnotation",
   "scala.annotation.MainAnnotation$",
 
-  //// New APIs: compiletime.ops
-  // Can be stabilized in 3.3.0 or later.
-  // Needs user feedback
-  "scala.compiletime.ops.any$.IsConst",
-  "scala.compiletime.ops.any$.ToString",
-  "scala.compiletime.ops.double", "scala.compiletime.ops.double$",
-  "scala.compiletime.ops.float",
-  "scala.compiletime.ops.float$",
-  "scala.compiletime.ops.int$.NumberOfLeadingZeros",
-  "scala.compiletime.ops.int$.ToDouble",
-  "scala.compiletime.ops.int$.ToFloat",
-  "scala.compiletime.ops.int$.ToLong",
-  "scala.compiletime.ops.long", "scala.compiletime.ops.long$",
-  "scala.compiletime.ops.string$.Length",
-  "scala.compiletime.ops.string$.Matches",
-  "scala.compiletime.ops.string$.Substring",
```

Comment:
@mbovel didn't we say that these operations would stay experimental?

Reply:
See https://github.com/lampepfl/dotty/pull/14431#issuecomment-1129030756; when we first discussed it, I hadn't realized that some compile-time operations were already stable. Given this, I re-discussed it with Martin and he agreed to make these operations stable as well.

The argument is that operations on `long`, `float` and `double` are exactly the same as the ones on `int` which are already stable, so it makes sense to make them stable as well. In term of maintenance, this doesn't change much.

Including operations on strings as well makes sense are they are important building blocks to do anything useful with strings at the type-level (Olivier used them for his work on regular expressions for example).